src/datasets/utils/download_manager.py

```diff
@@ -35,6 +37,11 @@
 from .info_utils import get_size_checksum_dict
 from .logging import get_logger
 from .py_utils import NestedDataStructure, map_nested, size_str
+from .streaming_download_manager import (
+    BASE_KNOWN_EXTENSIONS,
+    COMPRESSION_EXTENSION_TO_PROTOCOL,
+    _get_extraction_protocol_with_magic_number,
+)
```

Comment:
Would move this to `src/datasets/utils/file_utils.py` to avoid importing from `src/datasets/utils/streaming_download_manager.py` to `src/datasets/utils/download_manager.py`

Reply:
I am not sure if I understood well, do you want me to redefine the function and dictionnaries in `file_utils.py` or just move the imports to `file_utils.py`.
I used the second one in the commit 10cf70078fc34b5b5ba8c6f8afa945163329eb25 Until your response.

Reply:
> Would move this to `src/datasets/utils/file_utils.py` to avoid importing from `src/datasets/utils/streaming_download_manager.py` to `src/datasets/utils/download_manager.py`

Hello @mariosasko, moving **`COMPRESSION_EXTENSION_TO_PROTOCOL`** to `file_utils.py` creates a cyclic import error as it uses **`COMPRESSION_FILESYSTEMS`** which is imported from `filesystems.py`, and inside `filesystems.py` there are imports from `file_utils.py`.

How do I arrange import which is best for clarity and consistency ?

Reply:
Maybe let's move `COMPRESSION_EXTENSION_TO_PROTOCOL` to `filesystems.py` then?